packages/astro/src/preferences/defaults.ts

```diff
@@ -3,6 +3,12 @@ export const DEFAULT_PREFERENCES = {
 		/** Specifies whether the user has the Dev Overlay enabled */
 		enabled: true,
 	},
+	checkUpdates: {
+		/** Specifies whether the user has the update check enabled */
+		enabled: true,
+		/** Time since last check */
+		_lastCheck: 0,
```

Comment:
Is this the time **since** the last check or the timestamp **of** the last check?